invenio_app_ils/acquisition/jsonresolvers/order_order_lines.py

```diff
@@ -22,36 +23,38 @@ def jsonresolver_loader(url_map):
     """Resolve the referred document and patron for an Order Line."""
     from flask import current_app
 
-    def patron_resolver(order_line, patron):
+    def patron_resolver(order_line, patron_pid):
         """Resolve the Patron for the given Order Line."""
-        order_line["patron"] = patron.dumps_loader()
+        patron = get_patron_or_unknown(patron_pid)
+
+        order_line["patron"] = patron
```

Comment:
I know it's one line and convenient but this implicit modification here can lead to nasty errors...I would say maybe better rename that function to `order_line_patron_resolver` or `set_order_line_patron`. WDYT?

Reply:
@zzacharo what kind of errors do you mean? just for us to better remember this example
in general lets discuss this together